bin/libs/component.ts

```diff
@@ -494,6 +494,15 @@ export function processComponentApimlStaticDefinitions(componentDir: string): bo
           std.setenv('ZOSMF_NON_SECURE_PORT_ENABLED', `${nonSecurePortEnabled}`);
           std.setenv('ZOSMF_SECURE_PORT_ENABLED', `${securePortEnabled}`);
 
+          const zosmfAuthenticationScheme = ([ "zosmf", "httpBasicPassTicket" ].includes(std.getenv("ZOSMF_AUTHENTICATION_SCHEME"))) ? std.getenv("ZOSMF_AUTHENTICATION_SCHEME") : std.getenv("ZWE_zOSMF_authentication_scheme");
+          const authProvider = std.getenv("ZWE_components_apiml_apiml_security_auth_provider") ?? std.getenv("ZWE_components_gateway_apiml_security_auth_provider");
```

Comment:
Q: Should the same validation happen here?
Either stronger validation, or use `||` which will evaluate `""` to falsy 🙏 

```suggestion
          const authProvider = std.getenv("ZWE_components_apiml_apiml_security_auth_provider") || std.getenv("ZWE_components_gateway_apiml_security_auth_provider");
```

Reply:
It's not described in the schema as far as I can see, but the `ZWE_components_gateway_apiml_security_auth_provider` _must_ be set Zowe config, and to date we've guaranteed it's set through `defaults.yaml`. Still, stronger validation and an enumeration of acceptable values sounds like a good idea to me. I'll add this in if no one objects...